packages/treasury/bundles/install-on-chain.js

```diff
@@ -52,6 +53,7 @@ export async function installOnChain({ agoricNames, board, centralName, chainTim
     priceAuthority,
     loanParams,
     timerService: chainTimerService,
+    bootstrapPaymentValue: BOOTSTRAP_PAYMENT_VALUE,
```

Comment:
I will make this part of the options bag passed to `installOnChain`, but that's a simple matter of plumbing.  Maybe it will affect your tests, though, so it might be better to pass it as an optional property of the bag.

I will specify it in `bootstrap.js` when I need it.

Reply:
Yup, I can make it part of the options bag

Reply:
Done